### ISSUE TITLE
Request: [Using this tool to build a custom version of FFMPEG]

### ISSUE THREAD

**Issue by hnguyen48206** [m-ab-s/media-autobuild_suite#2017]
Hi,

In short, I came across this forked version of ffmpeg (https://github.com/numberwolf/FFmpeg-QuQi-H265-FLV-RTMP) which supports FLV (h265) encoding but do not really know how to use this tool with that src code to compile a custom build version of ffmpeg. After reading through the docs, it seems like this tool is meant for the default git src of ffmpeg, am I wrong? and if so, how do I actually carry out the process?

Best Regards.

**Comment by 1480c1** [comment]
> it seems like this tool is meant for the default git src of ffmpeg

you are correct


> how do I actually carry out the process

this is not supported by the suite, the best way to try to do this would be to use https://github.com/m-ab-s/media-autobuild_suite#custom-patches to checkout that repo instead such as

```bash
#!/bin/bash

_post_vcs() {
    git fetch https://github.com/numberwolf/FFmpeg-QuQi-H265-FLV-RTMP.git HEAD
    git reset --hard FETCH_HEAD
}
```

**Comment by 1480c1** [comment]
Your best option would be to try to request that he upstream his changes, although I'm not sure if it will be accepted or not as https://www.adobe.com/content/dam/acom/en/devnet/flv/video_file_format_spec_v10.pdf does not list h265 as a valid format and the closest format would be h264, so any flv file generated with h265 inside will probably be considered an invalid file.

Based on your issue in that repo, I would suggest instead to look into .ts (transport-streams) instead as those might fulfill your requirements

**Comment by GyanD** [comment]
Couple of times patches to support H.265 in FLV have been rejected, so don't count on that.

**Comment by hnguyen48206** [comment]
Thank you so much, both of you guys. The reason why I do not want to use HLS (h265) is because at this moment, there is still no way to be able to play that kind of stream on different kind of browsers which is a crucial part of streaming in the end. And I have came across a few JS players that claim to play FLV (h265), so I just want to give it a shot. 

One other thing is that I have found no where a re-compiled version of ffmpeg that does the trick.
